build.yaml

```diff
@@ -289,19 +307,37 @@ common_cli:
     ssh-add \$HOME/.ssh/id_rsa
 
   run_acceptance_aws_tests: &run_acceptance_aws_tests |
-    # we gather the return code, but make sure we come out of these tests with '0'
+    # we gather the return code but make sure we come out of these tests with 0
     # we store that code into RC and pass it to the end of the job execution.
-    parse_logs \$venv/bin/python admin/run-acceptance-tests --distribution \${DISTRIBUTION_NAME} --provider aws --dataset-backend aws --branch \${TRIGGERED_BRANCH} --build-server  http://\$(wget -qO- http://instance-data/latest/meta-data/public-ipv4)  --config-file /tmp/acceptance.yaml \${ACCEPTANCE_TEST_MODULE} ; RC=\$?
+    parse_logs \$venv/bin/python admin/run-acceptance-tests \
+    --distribution \${DISTRIBUTION_NAME} \
+    --provider aws --dataset-backend aws --branch \${TRIGGERED_BRANCH} \
+    --build-server  \
```

Comment:
hmmm,
the admin/run-acceptance-tests will provision his own flocker cluster, so those boxes need to download the rpm/debs from a repository.
on BB the omnibus pushes those pkgs to the buildbot master (or S3, can't recall), on Jenkins they only live on the slaves while they exist (releases are a different subject). That's why we point the acceptance test nodes to fetch their yum packages from a yum repo on the jenkins slave port 80.